index.bs

```diff
@@ -820,22 +814,38 @@ interface MLContext {
 <div class=internal-slots>
 {{MLContext}} has the following internal slots:
   <dl dfn-type=attribute dfn-for="MLContext">
-    : <dfn>\[[contextType]]</dfn> of type [=context type=]
+    : <dfn>\[[contextType]]</dfn> of type [=context type=].
     ::
         The {{MLContext}}'s [=context type=].
-    : <dfn>\[[deviceType]]</dfn> of type [=device type=]
+    : <dfn>\[[deviceType]]</dfn> of type {{MLDeviceType}}.
     ::
-        The {{MLContext}}'s [=device type=].
-    : <dfn>\[[powerPreference]]</dfn> of type [=power preference=]
+        The {{MLContext}}'s {{MLDeviceType}}.
+    : <dfn>\[[powerPreference]]</dfn> of type {{MLPowerPreference}}.
     ::
-        The {{MLContext}}'s [=power preference=].
+        The {{MLContext}}'s {{MLPowerPreference}}.
   </dl>
 </div>
 
+The <dfn>context type</dfn> is the type of the execution context that manages the resources and facilitates the compilation and execution of the neural network graph:
+<dl dfn-for="context type">
+<dt>"<dfn>default</dfn>"</dt>
+<dd>Context created per user preference options.</dd>
+<dt>"<dfn>webgpu</dfn>"</dt>
+<dd>Context created from WebGPU device.</dd>
+</dl>
+
 <div class="note">
 When the {{MLContext/[[contextType]]}} is set to [=context type/default=] with the {{MLContextOptions}}.{{MLContextOptions/deviceType}} set to {{MLDeviceType/"gpu"}}, the user agent is responsible for creating an internal GPU device that operates within the context and is capable of ML workload submission on behalf of the calling application. In this setting however, only {{ArrayBufferView}} inputs and outputs are allowed in and out of the graph execution since the application has no way to know what type of internal GPU device is being created on their behalf. In this case, the user agent is responsible for automatic uploads and downloads of the inputs and outputs to and from the GPU memory using this said internal device.
 </div>
 
+<dl dfn-type=dict-member dfn-for=MLComputeResult>
+    : <dfn>inputs</dfn>
+    :: An object where the keys are the graph input names, and the values are the bound {{ArrayBufferView}}s for the supplied input tensor values.
```

Comment:
Should we say "transferred {{ArrayBufferView}}s" instead? The `ArrayBufferView`s are the result of ["transfer an MLNamedArrayBufferViews" step](https://www.w3.org/TR/webnn/#mlnamedarraybufferviews-transfer) rather than the original user-bound ones.

Reply:
Sure, although #566 might change things.

Reply:
Done in 1f1b3c13762ec4366ebb9e233fec09f6ae432ddf